modules/offers/client/views/offer-host-edit.client.view.html

```diff
@@ -20,7 +20,11 @@
 
   <button type="submit"
           class="btn btn-lg btn-inverse-primary pull-right hidden-xs"
-          ng-disabled="offerHostEdit.isLoading"
+          ng-disabled="offerHostEdit.isLoading ||
+          (offerHostEdit.offer.status!=='no' && (offerHostEdit.offer.description|plainTextLength)<5)"
```

Comment:
There's quite a bit of repetition here with `<5` — let's move it to a method in a controller to consolidate it more?

Reply:
I tried it, but I couldn't make it work with $scope.$watch

Reply:
People here in collective also don't know, so either you tell me or I leave it as it is :)

Reply:
I stumbled upon a `$scope.$watch` that works. 83863861f265a6fa166e549c26865821d4826416
Still needs to get wired up with the other fields.
```js
$scope.$watch('offerHostEdit.offer.description', function (newValue) {
  console.log($filter('plainTextLength')(newValue));
});
```
I believe we were trying this together with @petr-hajek . I wonder why it didn't work.